site/content/en/docs/tasks/run/jobsets.md

```diff
@@ -32,7 +32,15 @@ metadata:
     kueue.x-k8s.io/queue-name: user-queue
 ```
 
-### b. Configure the resource needs
+### b. MultiKueue
+
+If the JobSet is submitted to a queue using [MultiKueue](/docs/concepts/multikueue) its `spec.managedBy` field needs to be set to `kueue.x-k8s.io/multikueue`.
+
+```yaml
+    alpha.jobset.sigs.k8s.io/managed-by: kueue
+```
```

Comment:
Oh I missed that, please drop or adjust.

Reply:
done